Gems/Multiplayer/Code/Include/Multiplayer/Components/NetworkCharacterComponent.h

```diff
@@ -53,6 +55,9 @@ namespace Multiplayer
         void OnActivate(Multiplayer::EntityIsMigrating entityIsMigrating) override;
         void OnDeactivate(Multiplayer::EntityIsMigrating entityIsMigrating) override;
 
+    protected:
+        void OnCharacterActivated() override;
```

Comment:
Maybe on `OnPhysicsCharacterActivated()` to make it intent clearer?

Reply:
I personally like the suggestion, but at least for now this naming is in line with the other physics bus names, so I'd prefer not to deviate. I'll leave it to the physics folks to decide if they want to make it more verbose in a follow-up PR.